docs/runtimes/application.ipynb

```diff
@@ -82,16 +82,306 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    application.deploy(with_mlrun=False)  # MLRun is already included in the requirements / image.
</pre><p>better to specify mlrun=True to be explicit. the default base image is python 3.9 which doesn't include mlrun</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/6168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>